packages/styles/src/variables/_color.scss

```diff
@@ -107,16 +107,6 @@ $background-colors: map.merge(
 $background-colors: map.merge($background-colors, $contextual-colors);
 $background-colors: map.merge($background-colors, $accent-colors);
 
-// One list to rule them all
-$colors: () !default;
-$colors: map.merge(
-  (
-    'black': $black,
-  ),
-  $grays
-);
-$colors: map.merge($background-colors, $colors);
-
```

Comment:
I'm not sure about this change, considering the comment for `packages/styles/src/themes/bootstrap/_overrides-color.scss` above.
Check if we need to reintegrate this to let utilites like `.text-primary` work again.

Reply:
Re-introduced minimal set of bootstrap colors